src/resources/postcss/tickets-admin-settings.pcss

```diff
@@ -27,4 +27,174 @@
+	font-style: normal;
+}
+
+/* The switch - the box around the slider */
+.tec-tickets-commerce-switch {
+    display: inline-block;
+    float: left;
+    height: 22px;
+    margin-right: 15px;
+    position: relative;
+    width: 40px;
+}
+
+/* Hide default HTML checkbox */
+.tec-tickets-commerce-switch input {
+	height: 0;
+	opacity: 0;
+	width: 0;
+}
+
+/* The slider */
+.tec-tickets-commerce-slider {
+	background-color: #ccc;
+	bottom: 0;
+	cursor: pointer;
+	left: 0;
+	position: absolute;
+	right: 0;
+	top: 0;
+	transition: .4s;
+	-webkit-transition: .4s;
+}
+
+.tec-tickets-commerce-slider:before {
+	background-color: white;
+	bottom: 4px;
+	content: "";
+	height: 26px;
+	left: 4px;
+	position: absolute;
+	transition: .4s;
+	width: 26px;
+	-webkit-transition: .4s;
+}
+
+.tec-tickets-commerce-slider.round {
+	border-radius: 34px;
+}
+
+.tec-tickets-commerce-slider.round:before {
+	border-radius: 50%;
+}
+
+.tec-tickets-commerce-slider:before {
+	background-color: white;
+	bottom: 3px;
+	height: 16px;
+	left: 3px;
+	position: absolute;
+	transition: .4s;
+	width: 16px;
+	-webkit-transition: .4s;
+}
+
+input:checked + .tec-tickets-commerce-slider {
+	background-color: #278DBC;
+}
+
+input:focus + .tec-tickets-commerce-slider {
+	box-shadow: 0 0 1px #278DBC;
+}
+
+input:checked + .tec-tickets-commerce-slider:before {
+	transform: translateX(18px);
+	-ms-transform: translateX(18px);
+	-webkit-transform: translateX(18px);
+}
+
+.tec-tickets-commerce-toggle {
+	margin-top: 45px;
+}
+
+.tec-tickets-commerce-toggle h2 {
+	font-size: 20px;
+	font-weight: bold;
+	line-height: 23px;
+	letter-spacing: 0.02em;
+}
+
+.tec-tickets-commerce-description {
+	font-size: 13px;
+	font-weight: normal;
+	max-width: 84%;
+	margin-bottom: 35px;
+}
+
+.tec-tickets-commerce-paypal {
+	background: #FFFFFF;
+	border: 1px solid #CCCCCC;
+	box-sizing: border-box;
+	border-radius: 4px;
+	display:flex;
+	flex-direction:row;
+	height: 325px;
+	justify-content: space-around;
+	padding: 46px;
+}
+
+.tec-tickets-commerce-paypal-logo img {
+	padding-right: 41px;
+}
+
+.tec-tickets-commerce-paypal-logo ul {
+	list-style: none;
+}
+
+.tec-tickets-commerce-paypal-logo li {
+	line-height: 1.8;
+}
+
+.tec-tickets-commerce-paypal-logo ul li::before {
+	color: #009BE1;
+	content: "\2022";
+	display: inline-block;
+	font-weight: bold;
+	width: 10px;
+	margin-left: -1em;
+}
+
+.tec-tickets-commerce-paypal-logo {
+	font-style: italic;
+	font-weight: normal;
+	font-size: 14px;
+	line-height: 16px;
+	letter-spacing: 0.02em;
+}
+
+.tec-tickets-commerce-paypal-connect {
+	max-width: 552px;
+}
+
+.tec-tickets-commerce-paypal-connect,
+.tec-tickets-commerce-paypal-logo {
+	display:flex;
+	flex-direction:column;
+}
+
+.tec-tickets-commerce-connect-paypal-button a {
+	background: #009BE1;
+	border-radius: 4px;
+	color: #FFFFFF;
+	font-weight: bold;
+	font-size: 16px;
+	line-height: 18px;
+	margin-top: 25px;
+	padding: 15px 30px;
+	position: absolute;
+	text-align: center;
+	text-decoration: none;
```

Comment:
```suggestion
	background: #009be1;
	border-radius: 4px;
	color: #fff;
	font-size: 16px;
	font-weight: bold;
	line-height: 18px;
	margin-top: 25px;
	padding: 15px 30px;
	position: absolute;
	text-align: center;
	text-decoration: none;
```

Reply:
All of the CSS above will be refactored by Juan before launch. But accepting the changes.